python.d/neo.chart.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+# Description:  netdata python.d module for monitoring Neo4j
+# Author: Jerome Baton, @wadael , copied from the example of Pawel Krupa (paulfantom)
+
+from bases.FrameworkServices.SimpleService import SimpleService
+
+try:
+    from neo4j.v1 import GraphDatabase
+    NEO4JDRIVER = True
+except ImportError:
+    NEO4JDRIVER = False
+
+
+ORDER = ['monnodes','monlabels','monrels']
+
+CHARTS = {
+    'monnodes': {
+        'options': [None, 'Nodes count', 'Nodes', 'Node count', 'neo4j', 'line'],
+        'lines': [
+             ['id', 'name', 'absolute', 1, 1, 'hidden']
+        ]
+    },
+    'monrels': {
+        'options': [None, 'Relations count', 'Nodes', 'Most present relations', 'neo4j', 'line'],
+        'lines': [
+             ['id', 'name', 'absolute', 1, 1, 'hidden']
+        ]
+    },
+    'monlabels': {
+        'options': [None, 'Relations count', 'Nodes', 'Most present labels', 'neo4j', 'line'],
+        'lines': [
+            ['id', 'name', 'absolute', 1, 1, 'hidden']
+        ]
+    }
+
+
+}
+
+class Service(SimpleService):
+
+    def __init__(self, configuration=None, name=None):
+        SimpleService.__init__(self, configuration=configuration, name=name)
+        self.order = ORDER
+        self.definitions = CHARTS
+        self.neosession = None
+        self.alive = True
+
+    def __exit__(self, exc_type, exc_value, traceback):
```

Comment:
Why is it here?

Reply:
I liked the idea to write a destructor again :)  
Also, saw this in some tut. I knew close to zero Python before that plugin of plugin.
Thought it was hygienic to have a destructor